roles/openshift_version/meta/main.yml

```diff
@@ -13,3 +13,5 @@ galaxy_info:
   - cloud
 dependencies:
 - role: lib_utils
+- role: container_runtime
```

Comment:
Please checkout my other patch to see the general direction that we're moving.

Container runtime will be setup as a prerequisite before openshift is deployed.  It cannot be a meta-dependency of any other role.